src/lib/features/feature-toggle/feature-toggle-service.ts

```diff
@@ -390,19 +390,25 @@ class FeatureToggleService {
         }
     }
 
-    validateConstraintsLimit(updatedConstrains: IConstraint[]) {
+    validateConstraintsLimit(constraints: {
```

Comment:
tbh I don't like that our validate functions are not private since it's tempting to call them from outside the service and I'd like to avoid that. On the other hand I understand the testability concern with fake stores setup.

Reply:
Addressed in 17ee7675bfa50707bbb8b14f363f77ba00d8cb91